### PR TITLE
show different claim states to Signator for Review

### PR DESCRIPTION
**Pull request by hobbitronics**
Signator:
image
Steward:
image

### CODE REVIEW

File: src/components/banners/ClaimBanner.svelte [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+import StatusBanner from '../banners/StatusBanner.svelte'
 import { getClaimState, SecondaryClaimStatus, State } from 'data/states'
 import type { ClaimStatus, ReceiptType } from 'data/claims'
-import StatusBanner from '../banners/StatusBanner.svelte'
+import type { UserAppRole } from 'data/user'
 
 export let claimStatus: ClaimStatus | SecondaryClaimStatus
 export let receiptType: ReceiptType
-export let isAdmin: boolean = false
+export let roleSelection: UserAppRole
```
Comment (Schparky) [inline src/components/banners/ClaimBanner.svelte:9]: There are other usages of ClaimBanner that probably need to be updated with this new property.

**Review by Schparky — CHANGES_REQUESTED** [review]
added one change request

File: src/components/banners/ClaimBanner.svelte (hunk 1 above)
Comment (hobbitronics) [reply src/components/banners/ClaimBanner.svelte:9]: I think I will just default to claimStates instead of requiring `roleSelection`

Comment (Schparky) [reply src/components/banners/ClaimBanner.svelte:9]: Does the roleSelection prop need a default? I can't remember if a warning or error is thrown when a component doesn't provide a default value and none is given by its consumer. I seem to remember a console warning.

**Review by RussellChamp — APPROVED** [review]
looks pretty good to me if you resolve schram's issues

**Comment by hobbitronics** [comment]
> added one change request

No build/console warnings now.

**Review by Schparky — APPROVED** [review]

**Pull request merged**